website/docs/docs/dbt-cloud-apis/schema-discovery-environment.mdx

```diff
@@ -16,6 +16,13 @@ The [example query](#example-query) illustrates a few fields you can query in th
 
 When querying for `environment`, you can use the following arguments.
 
+:::caution
+
+dbt Labs is making changes to the Discovery API to support multi-cell environment. These changes will take effect on September 7, 2023. 
+
+The value type `Int` for `id` is being deprecated and will be replaced with `BigInt`. Please update your API call accordingly to avoid API errors. 
```

Comment:
```suggestion
The value type `Int` for `id` is being deprecated and will be replaced with `BigInt`. When the time comes, you will need to update your API call accordingly to avoid errors. 
```